### DISCUSSION TITLE
User povisioing by using Cross-domain Identity Management (SCIM)

### DISCUSSION THREAD

**Discussion by bkhashfeh** [keycloak/keycloak#8765] [Q&A] [answered]
Hi keycloackers,
we are in a process to go live with KeyCloak as an offical Idp for all of our applications, as well as internal applications (around 16).

There is a requirement from the digital transformation team to support User provisioning, so we can add\delete user from KeyCloak, and it will be reflected automatically to all of our internal applications like G-Suite, JIRA, Slack, ect.

I know that KeyCloak yet to support SCIM, [but it is in the roadmap ](https://issues.redhat.com/browse/KEYCLOAK-2537)

Is there any clear date on when we can see it live?

**Comment by kkcmadhu** [comment]
i too look forward to this feature keenly.

Since a major revamp is happening int he storage layer, i think it will be the right time to build something like this from grounds up in new storage.

**Reply by hrenard** [reply]
In the meantime, we're working on an extension : https://lab.libreho.st/libre.sh/scim/keycloak-scim. Caution, It isn't yet polished.

**Comment by schuerg** [comment]
Is SCIM support still on the roadmap?

**Comment by jonkoops** [comment, marked as answer]
Support can be tracked under #13484
